releasenotes/notes/0.22/add-barrier-label-8e677979cb37461e.yaml

```
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    Added a ``label`` parameter to the :class:`.Barrier` class's constructor
```

Not sure if the `s` goes here, but maybe an english native-speaker could check! 🙂 
```suggestion
    Added a ``label`` parameter to the :class:`.Barrier` class' constructor
```

---

Either way is actually ok, I learned how to write in school from the "[The Elements of Style"](https://en.wikipedia.org/wiki/The_Elements_of_Style) which clearly says use `'s`: https://gutenberg.org/files/37134/37134-h/37134-h.htm#Rule_1 but either way is acceptable and both are used. So I'll defer to what you prefer